src/components/storefront/CartItem/CartItem.js

```diff
@@ -55,16 +70,24 @@ const CartItem = ({ title, sku, image, price, quantity, lineItemId, checkoutId }
           {/* Does not show colour + variant at the moment */}
         </VStack>
       </Flex>
-      <QuantityPicker size="checkout" {...quantityPickerProps} />
-      <IconButton
-        size="xs"
-        color="brand.gray"
-        variant="unstyled"
-        aria-label="Remove from cart"
-        icon={<CloseIcon />}
-        onClick={removeFromCart}
-      />
-    </Flex>
+      <Stack
+        justifyContent="space-between"
+        align={{ base: 'flex-end', lg: 'flex-start' }}
+        direction={{ base: 'column-reverse', lg: 'row' }}
+        spacing={[2, 6]}
+        h="100%"
+      >
+        <QuantityPicker size="checkout" {...quantityPickerProps} />
+        <IconButton
+          size="xs"
+          color="brand.gray"
+          variant="ghost"
+          aria-label="Remove from cart"
+          icon={<CloseIcon />}
+          onClick={removeFromCart}
+        />
+      </Stack>
```

Comment:
I think I remember the designers wanted the space between the X and the quantity picker to be big, you should probably make sure with them that they're okay with this spacing here: 
![image](https://user-images.githubusercontent.com/24748963/116008405-5c9e4400-a5e2-11eb-950a-7129578cf3cf.png)